### ISSUE TITLE
Hyperelliptic curves that realize isogeny class over Fq as Jacobian have defining polynomials swapped

### ISSUE THREAD

**Issue by edgarcosta** [LMFDB/lmfdb#4730]
For example, the curves listed in
https://www.lmfdb.org/Variety/Abelian/Fq/4/2/a_a_ac_a
has genus 8, while the right curve is
```
sage: R.<x> = GF(2)[]
sage: C = HyperellipticCurve((x^9+x^7+x^3+1), 1)
sage: C
Hyperelliptic Curve over Finite Field of size 2 defined by y^2 + y = x^9 + x^7 + x^3 + 1
sage: C.genus()
4
sage: C.frobenius_polynomial().reverse()
16*x^8 - 4*x^5 - 2*x^3 + 1
```

**Comment by edgarcosta** [comment]
@roed314 the curves data is stored as strings, see https://www.lmfdb.org/api/av_fq_isog/?label=4.2.ad_f_aj_p, should one regenerate the data or fix the data in place?

**Comment by kedlaya** [comment]
Assuming this is consistent (which it appears to be, I just bumped into this in some other examples), surely we can just fix the data in place?

**Comment by roed314** [comment]
Yeah, I'm sorry to have been slow to get around to this.  I've been distracted by groups.  I'll take a look on Monday.

**Comment by roed314** [comment]
The switch was in the original file I got for genus 4 over F2.  I've reversed the polynomials in the hyperelliptic case and updated the table.
